app/models/qing_group.rb

```diff
@@ -1,9 +1,12 @@
 class QingGroup < FormItem
   include Translatable
 
-  translates :group_name
+  translates :group_name, :group_hint
 
   replicable child_assocs: :children, backward_assocs: :form, dont_copy: [:form_id]
 
   alias_method :c, :children
+
+  scope(:child_groups, ->(children) { children.where(type: "QingGroup").all })
```

Comment:
I don't think you want the `.all` here. Typically you don't put that in a scope. I think it actually has no effect.